sdks/java/core/src/main/java/org/apache/beam/sdk/io/WriteFiles.java

```diff
@@ -122,27 +126,35 @@
    * Creates a {@link WriteFiles} transform that writes to the given {@link FileBasedSink}, letting
    * the runner control how many different shards are produced.
    */
-  public static <T> WriteFiles<T> to(FileBasedSink<T> sink) {
+  public static <InT, DestT, OutT> WriteFiles<InT, DestT, OutT> to(
```

Comment:
I think I was just trying to keep this on one line - but I'll change to use consistent terminology. (also at some point this was InputT, and I renamed it).